CONTRIBUTING.md

```diff
@@ -15,28 +15,55 @@ No software is bug free. So, if you got an issue, follow these steps:
   * Share error logs, screenshots and etc.
   * To speed up the issue fixing process, send us a sample repo with the issue you faced:
 
+### Testing against `master`
+
+To test your project against the current latest version of storybook, you can clone the repository and link it with `npm`. Try following these steps:
+
+1. Download the latest version of this project, and build it
+
+  ```
+  git clone https://github.com/storybooks/storybook.git
+  cd storybook
+  npm install
```

Comment:
At this moment when I `npm install` it only installs root dependencies. In order to install all packages, I need to run `npm run bootstrap`.
OS: Windows. So I can't tell is it common or only windows issue (but previosly `npm install` started to install everything inside packages folder)